app/views/tickets/_form.html.erb

```diff
@@ -55,7 +55,7 @@
   <%= render 'attachments/form', f: f %>
 
   <p>
-    <%= f.submit class: 'button regular radius' %>
+    <%= f.submit t(:create_ticket), class: 'button regular radius' %>
```

Comment:
This is not needed, it is automatically derived by Rails. By adding this you break all other translations. Could you undo this change? I can then merge the rest. Thanks a lot!